code/src/components/Name.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+
+
+
+
+
+ const Name = ({userName, onInputChange, onNextChange}) => {
+    return (
+        <section className="container">
+          
+        <h2>Thank you for participating!</h2>
+        <label htmlFor="name">Please type your name</label>
+        
+        <input
+        id="name" 
+        type="text"
+        value={userName}
+        onChange={onInputChange}/>
+        <button disabled={userName === ''} onClick={onNextChange} className="btn">Next</button>
```

Comment:
It´s possible to click the button even though there is no input, so I think this disabled-function doesn´t work. I´m not sure why though, maybe it´s something simple as removing the spaces?